.github/workflows/push.yml

```diff
@@ -41,6 +44,10 @@ jobs:
         uses: ./.github/actions/setup-oracle
       - name: apt
         uses: ./.github/actions/apt-x64
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1.2
+        with:
+          key: ${{github.job}}-${{matrix.debug}}-${{matrix.zts}}
```

Comment:
can a job name be used here? if not, you can use https://docs.github.com/en/actions/learn-github-actions/expressions#tojson

also the cache key should contain the minor PHP version/branch, you can use hash of https://github.com/php/php-src/blob/master/main/php_version.h file

Reply:
Will do! (This evening or so.)

Reply:
I wonder if this should contain the branch name? This is targetting master but it would make sense to backport. In that case the cache key should be different to avoid invalidating the cache of other branches.

Reply:
2x yes - master will be branch cutted someday and PHP-8.1/8.2 CI will benefit from this PR as well